app/views/templates/news_article.html.haml

```diff
@@ -3,26 +3,30 @@
 
 - breadcrumb :public_news_article, node
 
-%div.list-horizontal
-  = render partial: 'templates/related_sections', locals: {node: node}
-
 %h1
   =node.name
 
 %div.abstract
   %p
     =node.short_summary
     - if node.summary
-      != markdown_content node.summary
+      .summary
+        != markdown_content node.summary
 
 %ul.announcement-info
   - if node.release_date
-    %li Published date: #{l(node.release_date, format: :news)}
+    %li
+      %span
+        Date published
+      #{l(node.release_date, format: :news)}
   %li
-    Published by:
-    %a= link_to(node.section.name, public_node_path(node.section.home_node))
-
+    %span
+      Published by
+    %a= link_to(node.section.name, section_news_articles_path(section: @section.home_node.slug))
 
 = render partial: 'templates/content_body', object: node.content_body
 
 =@hierarchy
+
+-#%div.list-horizontal
+  = render partial: 'templates/related_sections', locals: {node: node}
```

Comment:
Tags are out for MVP of this page

Having related articles at the end of the document makes sense to me as it seems that users would naturally be looking for related articles after reading the content. In usability testing sessions related articles were placed in this position and tested well on desktop and mobile. 

I would say for MVP this is a fine approach but its an interesting question. I'll let Sam know and we might even test this idea next week in the prototype.

Any opinions @klepas